### ISSUE TITLE
fetch(new Request()) throws a TypeError fetch-mock: Unrecognised Request object

### ISSUE THREAD

**Issue by jmc-rival** [wheresrhys/fetch-mock#437]
I am using fetch-mock 7.3.3 with @types/fetch-mock 7.3.1 (not sure why these versions are different) and I was getting an error similar to the one raised in this closed issue https://github.com/wheresrhys/fetch-mock/issues/156

Reading that other thread, I was able to fix it by modifying the proposed solution that wraps the fetchHandler. 

```
const fetchMock = require('fetch-mock');

const _fetchMock = fetchMock.fetchHandler;
fetchMock.fetchHandler = (url, options) => {
  if (url instanceof Request && options === undefined) {
    // Function was called with a single argument of type Request
    return _fetchMock(url.url, { method: url.method });
  }
  return _fetchMock(url, options);
};
```

I am using TypeScript with Node and Jest

**Comment by wheresrhys** [comment]
Could you see if this is fixed in v8.0.0-alpha.9? @jmc-rival

**Comment by wheresrhys** [comment]
closing due to inactivity
